lm_eval/tasks/jsquad.py

```diff
@@ -46,14 +46,14 @@ def _squad_agg(key, item):
 
 class JSQuAD(Task):
     """
-    prompt format refered to [日本語に特化した60億パラメータ規模のGPTモデルの構築と評価](https://www.anlp.jp/proceedings/annual_meeting/2023/pdf_dir/H9-4.pdf)
+    prompt template is taken from [日本語に特化した60億パラメータ規模のGPTモデルの構築と評価](https://www.anlp.jp/proceedings/annual_meeting/2023/pdf_dir/H9-4.pdf)
     """
-    VERSION = 1.1
+    # VERSION = {structure_version}.{prompt_template_version}
+    VERSION = 0.1
```

Comment:
Could we change the format to `{task_name}-{structure_ver}-{prompt_tpl_ver}`? E.g., change the `.` separator to `-`. 

And I think we should call the `structure_version` as `data_version` and in all tasks, we should keep `1.1` as data version for all JGLUE tasks b/c that's how Yahoo is versioning their data:

- https://github.com/yahoojapan/JGLUE/tree/main/datasets
![image](https://github.com/Stability-AI/lm-evaluation-harness/assets/3454980/38352c96-8234-42bd-88ad-45a2ae1a1596)


So for the default JSQuAD task, its name will become
- `jsquad-1.1-0.1`

And for the Fintan class, it will be 
- `jsquad-1.1-0.2`

Same rule apply to other JGLUE tasks.